python/lsst/obs/subaru/isr.py

```diff
@@ -300,10 +297,15 @@ def runDataRef(self, sensorRef):
             flatExposure = self.getIsrExposure(sensorRef, "flat")
             self.flatCorrection(ccdExposure, flatExposure)
 
+        if self.config.doDefect:
+            self.maskAndInterpDefect(ccdExposure, defects)
+        if doWriteDebug: ccdExposure.writeFits(outDir+"/isr7.fits")
```

Comment:
Yep, it was just a relic from when I was comparing isr'd exposures at every stage.